src/service/url-replacements-impl.js

```diff
@@ -163,6 +164,18 @@ export class UrlReplacements {
       return removeFragment(info.sourceUrl);
     }));
 
+    this.setAsync_('SOURCE_URL', () => {
```

Comment:
This is added. I didn't change the previous `this.set_` https://github.com/zhouyx/amphtml/blob/366a535bf7a4c0c0e33125637c1052593a5bd199/src/service/url-replacements-impl.js#L201
it's still there.